src/Microsoft.Azure.ServiceBus/Primitives/ServiceBusConnection.cs

```diff
@@ -42,6 +42,11 @@ protected ServiceBusConnection(TimeSpan operationTimeout, RetryPolicy retryPolic
         /// </summary>
         public string SasKeyName { get; set; }
 
+        /// <summary>
+        /// Get the transport type from the connection string
```

Comment:
I know this is optional, but having more information on XML doc is such an experience improvement. When intellisense tells you what those are, it save extra operation. Perhaps a \<remarks\> to say AMQP and WebSockets.